src/pubsub/mod_pubsub.erl

```diff
@@ -822,6 +861,7 @@ out_subscription(Acc, User, Server, JID, subscribed) ->
 out_subscription(Acc, _, _, _, _) ->
     Acc.
 
+%% EXPORTED FOR REAL
```

Comment:
what does it mean ?

Reply:
this is me committing too much. I was just telling myself which functions are really part of the external API and I shouldn't touch

Reply:
Ok, can we remove it ?